foundation/container.go

```diff
@@ -216,7 +215,7 @@ func (c *Container) MakeRateLimiter() httpcontract.RateLimiter {
 }
 
 func (c *Container) MakeRoute() routecontract.Engine {
-	instance, err := c.Make(route.Binding)
```

Comment:
S: I think we'd better to keep the `route.Binding` constant, it can be used by other packages.